### PR TITLE
PROD-2122 Redirect to work dashboard when user is log in -> Dev

### PR DESCRIPTION
**Pull request by hentrymartin**
What's this PR is about?

I want to discuss in the PR whether we need to fix this url. 

Context:

In login-prompt component, the fallback url is hardcoded to "/self-service/branding" which is the old branding url. 

There are two scenario's here, 

1. When user is log in
2. When user is not log in

Scenario 1(When user is log in):

1. Going to this url(https://platform.topcoder-dev.com/self-service/login-prompt) redirects to the old branding page(https://platform.topcoder-dev.com/self-service/branding)

Scenario 2(When user is not log in):
1. Going to login prompt url(https://platform.topcoder-dev.com/self-service/login-prompt) does not redirect but displays the login prompt page(which is correct I believe).

@brooketopcoder Personally I think the scenario 1 is not valid since the user won't be going to login-prompt if they are already log in. Do I have to fix the url here? Am I missing something else because of my restricted knowledge in existing functionality?

Thanks!

### CODE REVIEW

File: src/routes/LoginPrompt/index.jsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ const LoginPrompt = ({
 
   useEffect(() => {
     if (isLoggedIn) {
+      // What should be the change here?
       navigate(nextPageUrl || "/self-service/branding");
```
Comment (hentrymartin) [inline src/routes/LoginPrompt/index.jsx]: @brooketopcoder Please refer to the PR description. But the question in short is do we need to change the url here?

Comment (brooketopcoder) [reply src/routes/LoginPrompt/index.jsx]: @hentrymartin I think we can redirect back to the work dashboard if the user is already logged in.

Granted, someone shouldn't be directed to the login prompt if they're already logged in, but if they have it bookmarked or something, we should still take them to the dashboard instead.

Comment (hentrymartin) [reply src/routes/LoginPrompt/index.jsx]: ok thanks. agreed. I've changed the url to work dashboard.

File: src/routes/LoginPrompt/index.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const LoginPrompt = ({
 
   useEffect(() => {
     if (isLoggedIn) {
-      navigate(nextPageUrl || "/self-service/branding");
+      navigate(nextPageUrl || "/work/dashboard");
```
Comment (brooketopcoder) [inline src/routes/LoginPrompt/index.jsx]: plz use the global constant for this route:  ROUTES.DASHBOARD_PAGE

**Review by brooketopcoder — CHANGES_REQUESTED** [review]

File: src/routes/LoginPrompt/index.jsx (hunk 2 above)
Comment (hentrymartin) [reply src/routes/LoginPrompt/index.jsx]: changed 👍🏻

**Review by brooketopcoder — APPROVED** [review]

**Review by testflyjets — APPROVED** [review]

**Pull request merged**